appveyor.yml

```diff
@@ -113,7 +142,8 @@ build_script:
 
   # install the wheel
   - ps: python -m pip install --upgrade pip
-  - ps: python -m pip install (gci dist\*.whl | % { "$_" })
+  - ps: python -m pip install --no-deps --ignore-installed (gci dist\*.whl | % { "$_" })
```

Comment:
Would it make sense to drop the --upgrade in .travis.yml and also use --no-deps --ignore-installed?